pkg/peertls/extensions.go

```diff
@@ -4,13 +4,62 @@
 package peertls
 
 import (
+	"bytes"
+	"crypto"
 	"crypto/x509"
 	"crypto/x509/pkix"
 	"encoding/asn1"
+	"encoding/binary"
+	"encoding/gob"
+	"time"
 
 	"github.com/zeebo/errs"
 
-	"storj.io/storj/pkg/utils"
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/boltdb"
+	"storj.io/storj/storage/redis"
+)
+
+const (
+	// SignedCertExtID is the asn1 object ID for a pkix extensionHandler holding a
+	// signature of the cert it's extending, signed by some CA (e.g. the root cert chain).
+	// This extensionHandler allows for an additional signature per certificate.
+	SignedCertExtID = iota
+	// RevocationExtID is the asn1 object ID for a pkix extensionHandler containing the
+	// most recent certificate revocation data
+	// for the current TLS cert chain.
+	RevocationExtID
+	// RevocationBucket is the bolt bucket to store revocation data in
+	RevocationBucket = "revocations"
+)
+
+const (
```

Comment:
why separate the `const` declarations ?

Reply:
to reset iota... I liked this instead of `iota = 0`